manifest/provider/plan.go

```diff
@@ -353,23 +351,20 @@ func (s *RawProviderServer) PlanResourceChange(ctx context.Context, req *tfproto
 	s.logger.Debug("[PlanUpdateResource]", "OAPI type", dump(so))
 
 	// Transform the input manifest to adhere to the type model from the OpenAPI spec
-	morphedManifest, err := morph.ValueToType(ppMan, objectType, tftypes.NewAttributePath())
-	if err != nil {
-		resp.Diagnostics = append(resp.Diagnostics, &tfprotov5.Diagnostic{
-			Severity: tfprotov5.DiagnosticSeverityError,
-			Summary:  "Failed to morph manifest to OAPI type",
-			Detail:   err.Error(),
-		})
+	morphedManifest, d := morph.ValueToType(ppMan, objectType, tftypes.NewAttributePath().WithAttributeName("object"))
+	if len(d) > 0 {
+		resp.Diagnostics = append(resp.Diagnostics, d...)
 		return resp, nil
 	}
 	s.logger.Debug("[PlanResourceChange]", "morphed manifest", dump(morphedManifest))
 
-	completePropMan, err := morph.DeepUnknown(objectType, morphedManifest, tftypes.NewAttributePath())
+	completePropMan, err := morph.DeepUnknown(objectType, morphedManifest, tftypes.NewAttributePath().WithAttributeName("object"))
 	if err != nil {
 		resp.Diagnostics = append(resp.Diagnostics, &tfprotov5.Diagnostic{
-			Severity: tfprotov5.DiagnosticSeverityError,
-			Summary:  "Failed to backfill manifest from OAPI type",
-			Detail:   err.Error(),
+			Severity:  tfprotov5.DiagnosticSeverityError,
+			Summary:   "Failed to backfill manifest from OAPI type",
```

Comment:
I would say this error message probably isn't going to mean much to the user. 

Are we able to formulate something actionable here like `This usually happens when _________`?